scripts/projectView.js

```diff
@@ -15,9 +15,8 @@
       $('#projects').append(a.toHtml('project'));
     });
     projectView.handleMainNav();
-    Project.wordCount = Project.prototype.findDescWordCount();
-    console.log(Project.prototype.wordCount);
-    $('#fun-facts').append(Project.prototype.toHtml('word-count'));
+    Project.all[0].wordCount = Project.findDescWordCount();
+    $('#fun-facts').append(Project.all[0].toHtml('word-count'));
   };
```

Comment:
Is this, strictly, kosher? The big thing I ran into was that my .toHtml() function was attached to the Project prototype (and needed to be for its original intended use (with 'project') to work)--So even keeping findDescWordCount as a declared method  on the Project constructor (rather than the prototype), I still ultimately had to attach the total word count to a single instance in the Project.all array and call .toHtml off that. Which is the greater sin? Doing that (which makes it less functionally arranged I should think) or repeating myself by making a similar .toHtml function on Project as well as the prototype?